test/integration/replicaset/test_replicaset_vshard.py

```diff
@@ -0,0 +1,236 @@
+
+    status_cmd = [tt_cmd, "rs", "vs", "bootstrap", "test_custom_app:unexist"]
+    rc, out = run_command_and_get_output(status_cmd, cwd=tmpdir_with_cfg)
+    assert rc == 1
+    assert re.search(r"   ⨯ instance \"unexist\" not found", out)
+
+
+@pytest.mark.skipif(tarantool_major_version > 2,
+                    reason="skip custom test for Tarantool > 2")
+@pytest.mark.parametrize("flag", [None, "--custom"])
+def test_vshard_bootstrap_custom_app(tt_cmd, tmpdir_with_cfg, flag):
+    tmpdir = tmpdir_with_cfg
+    app_name = "test_custom_app"
+    app_path = os.path.join(tmpdir, app_name)
+    shutil.copytree(os.path.join(os.path.dirname(__file__), app_name), app_path)
+    try:
+        # Start a cluster.
+        start_cmd = [tt_cmd, "start", app_name]
+        rc, out = run_command_and_get_output(start_cmd, cwd=tmpdir)
+        assert rc == 0
+
+        # Check for start.
+        file = wait_file(os.path.join(tmpdir, app_name), 'ready', [])
+        assert file != ""
+
+        cmd = [tt_cmd, "rs", "vs", "bootstrap"]
+        if flag:
+            cmd.append(flag)
+        cmd.append("test_custom_app:test_custom_app")
+
+        rc, out = run_command_and_get_output(cmd, cwd=tmpdir)
+        assert rc == 1
+        assert re.search(r"""  • Discovery application...*
+
+Orchestrator:      custom
+Replicasets state: bootstrapped
+
+• .*
+  Failover: unknown
+  Master:   single
+    • test_custom_app .* rw
+
+   • Bootstrapping vshard.*
+   ⨯ bootstrap vshard is not supported for an application by "custom" orchestrator
+""", out)
+    finally:
+        stop_cmd = [tt_cmd, "stop", app_name]
+        rc, _ = run_command_and_get_output(stop_cmd, cwd=tmpdir)
+        assert rc == 0
+
+
+@pytest.mark.skipif(tarantool_major_version > 2,
+                    reason="skip cartridge test for Tarantool > 2")
+@pytest.mark.parametrize("flag", [None, "--cartridge"])
+@pytest.mark.parametrize("target", ["uri", "app", "instance"])
+@pytest.mark.parametrize("cartridge_app", [{"bootstrap_vshard": False}], indirect=True)
+def test_vshard_bootstrap_cartridge(cartridge_app, tt_cmd, target, flag):
+    cmd = [tt_cmd, "rs", "vs", "bootstrap"]
+    if flag:
+        cmd.append(flag)
+    if target == "uri":
+        cmd.extend(["--username", cartridge_username, "--password", cartridge_password])
+        cmd.append(cartridge_app.uri["s1-replica"])
+    elif target == "app":
+        cmd.append(cartridge_name)
+    elif target == "instance":
+        cmd.append(f"{cartridge_name}:s1-replica")
+
+    rc, out = run_command_and_get_output(cmd, cwd=cartridge_app.workdir)
+    assert rc == 0
+    buf = io.StringIO(out)
+    assert "• Discovery application..." in buf.readline()
+    buf.readline()
+    # Skip init status in the output.
+    parse_status(buf)
+    assert "Bootstrapping vshard" in buf.readline()
+    assert "VShard bootstrapping is done. Check the cluster status" in buf.readline()
```

Comment:
Why do we need a message about checking the cluster status?

Reply:
@oleg-jukovec this comment relates to the [your comment about ](https://github.com/tarantool/tt/pull/820#discussion_r1574414404) messages.

It's a cartridge specific. Begins from some version it has a partial bootstrap: when you bootstrap only one of the several groups, as I understood from my investigation. And tool says to you: "it's possibly that we do the partial bootstrap, whereas you ask for the full, so, check the status".

For the details please check this https://github.com/tarantool/cartridge-cli/commit/5619178ce1cf13e2674f33e22d31eccec8cc1e1b and the mentioned issue.

Reply:
I think we need the same messages in both cases.

Reply:
The message is unified to the minimalistic "Done.". "Check cluster status" is removed, because it's expected behavior that cartridge can do partial bootstrap.